dracut/30ignition/module-setup.sh

```diff
@@ -36,6 +36,10 @@ install() {
     # This one is optional; https://src.fedoraproject.org/rpms/ignition/pull-request/9
     inst_multiple -o mkfs.btrfs
 
+    # Required by s390x's z/VM installation.
+    # Supporting https://github.com/coreos/ignition/pull/865
+    inst_multiple -o chccwdev vmur tar
```

Comment:
I think @ajeddeloh is trying hard to avoid having `tar` here. I do agree it would be beneficial to get a single Ignition config file (JSON blob) as the spool file, only.

Reply:
I did not follow all the development in the coreos/ignition#865, just trying to help where it's needed. Will update when there is a decision is made there.

Reply:
@tuan-hoang1 Thank you for this support. I have removed the tar code in  coreos/ignition#865 , so Tar is not necessary here. Please update here.

Reply:
Updated, thanks.